src/components/CreateEventForm/CreateEventForm.tsx

```diff
@@ -154,6 +164,7 @@ export default function CreateEventForm(): JSX.Element {
                                         onChange={setEndTimeValue}
                                         name="endTime"
                                         allowEmpty={false}
+                                        use12Hours={!userRecord?.timeFormat24Hr}
```

Comment:
I assume this is negated because it is undefined? If we stick with 12-hour as the default, this negation could be removed.

Reply:
It's negated because the default is 24 hour. Maybe I misunderstood, but I thought that's what Amy and I agreed on.